torch/_functorch/aot_autograd.py

```diff
@@ -4541,7 +4541,7 @@ def functional_call(named_params, named_buffers, *args, **kwargs):
     named_buffers = dict(mod.named_buffers(remove_duplicate=False))
     num_params_buffers = len(named_params) + len(named_buffers)
     compiled_f = aot_function(
-        functional_call, num_params_buffers=num_params_buffers, *args, **kwargs
+        functional_call, *args, **kwargs, num_params_buffers=num_params_buffers
```

Comment:
Likewise, do we expect users to be able to override this?

Reply:
Maybe we should add an assert that the specific keys is not kwargs.

Reply:
Same as above